libcontainer/cgroups/fs2/fs2.go

```diff
@@ -76,8 +76,8 @@ func (m *manager) Apply(pid int) error {
 		// - "runc create (rootless + limits + no cgrouppath + no permission) fails with informative error"
 		if m.rootless {
 			if m.config.Path == "" {
-				cl, clErr := neededControllers(m.config)
-				if clErr == nil && len(cl) == 0 {
+				cl, clErr := needAnyControllers(m.config)
```

Comment:
nit: `cl` used to mean "controller list". Now it's not so maybe rename to `need` or something.

ditto for `clErr`.